### PR TITLE
Project gutenburg texts

### PR DESCRIPTION
**Pull request by rah-ds**

### CODE REVIEW

File: download_other_books.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!usr/bin/python3
```
Comment (EfrainOlivaresUVA) [inline download_other_books.py:1]: This is good and I use `os.system` a bunch too.  Just for learning purposes, what would the bash version of the loop look like?

File: makefile [hunk 2]
```diff
@@ -0,0 +1,42 @@
+default:
+	@cat makefile
+
+#Download Raven only
+get_raven_data:
+	mkdir -p works/
```
Comment (EfrainOlivaresUVA) [inline makefile:6]: cool, the -p is often overlooked...

File: makefile [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+	echo "raven matches"
+	cat works/raven.txt | grep "raven" | wc -w
+	echo "Raven matches"
+	cat works/raven.txt | grep "Raven" | wc -w
+	echo "Raven or raven - matches"
+	cat works/raven.txt | grep -E "raven|Raven" | wc -w
+
+#get the other files
+get_rest_of_texts:
+	#already ran chmod 777, couldn't figure out bash script
```
Comment (EfrainOlivaresUVA) [inline makefile:28]: what was the error you ran into?

File: makefile [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+#Total all files -- including the raven -- line counts
+total_lines works/:
+	bat works/*.txt | wc -l
+
+#Total all works - word count
+total_words:
+	bat works/*.txt | wc -w
+
+#clean up
+clean_up:
```
Comment (EfrainOlivaresUVA) [inline makefile:40]: ah ya, the clean up is always good a good idea

**Review by EfrainOlivaresUVA — APPROVED** [review]

**Review by MiaHanzhangYuan — APPROVED** [review]

**Comment by MiaHanzhangYuan** [comment]
Nice work! (Type the branch name correctly, though.)

**Pull request merged**